fastlane/Fastfile

```diff
@@ -112,9 +112,11 @@ platform :ios do
 
     ios_bump_version_release(skip_deliver: true)
     new_version = ios_get_app_version
-    extract_release_notes_for_version(version: new_version,
-      release_notes_file_path: File.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
-      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'))
+    extract_release_notes_for_version(
+      version: new_version,
+      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
+      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt')
+    )
```

Comment:
This is just formatting and unrelated to the change, sorry for the noise. My IDE kept correcting it so I figured I'd leave it there.

Reply:
This is the right kind of formatting. Thank you for tracking it so future users won't have to 👍